### ISSUE TITLE
[bug] Invalid argument(s) when setting the preferredCropAspectRatio to less than 1(for example - 9 /16 ratio)

### ISSUE THREAD

**Issue by LeayAb** [LeGoffMael/video_editor#89]
![image](https://user-images.githubusercontent.com/39321031/165207469-944f3774-6e7c-4c83-a98d-e117ec9bfddc.png)

**Comment by LeGoffMael** [comment]
Thanks for the report, i will check it when i have some times

**Comment by GelidGeorge** [comment]
Videos has to be exported vertically in the project I am working on. Could you optimise the crop handles logic a bit more so that. The Preferred Aspect Ratio is always used for cropping a video. Some times the ratio are ignore. When the top handle is dragged down while the bottom crop handle is at the bottom. Preferred aspect ratio kind of breaks...

**Comment by LeGoffMael** [comment]
Could you test using this branch -> https://github.com/seel-channel/video_editor/tree/fix-vertical-crop

```yaml
video_editor:
  git:
    url: https://github.com/seel-channel/video_editor
    ref: fix-vertical-crop
```

And let me know if you're still facing an issue ?

**Comment by novvia-dev** [comment]
@LeGoffMael tested that fork, but it needs to be merged back into develop / updated, many out of date errors.

lib/shared_widgets/video_cropper.dart:85:45: Error: The argument type 'File?' can't be assigned to the parameter type 'File' because 'File?' is nullable and 'File' isn't.
 - 'File' is from 'dart:io'.
          appState.mediaFileStream.addMedia(file, position: 0);
                                            ^
lib/shared_widgets/video_cropper.dart:78:7: Error: No named parameter with the name 'onError'.
      onError: (e, s) => _exportText = 'Failed. Try again.', //'Error on export video :(',
      ^^^^^^^

**Comment by rahuldubey11** [comment]
Please fix this issue in the main branch.

**Comment by LeGoffMael** [comment]
Fixed in 1.5.1
